### PR TITLE
Mag 14

### PR DESCRIPTION
**Pull request by BronzePointKrel**
Hello Liz. The majority of changes are with Data/Reach.php

Let me know if you have any questions.

### CODE REVIEW

File: Helper/Data.php [hunk 1]
```diff
@@ -156,67 +199,77 @@ public function canAllowMultiPleCurrency()
     }
 
     /**
-     * Get currencly allowed for specific countries config
+     * Get currently allowed for specific countries config  REACH_SPECIFIC_PATH - Function 07
```
Comment (lizzyWithReach) [inline Helper/Data.php]: should be currency, not currently

File: Helper/Data.php [hunk 2]
```diff
@@ -156,67 +199,77 @@ public function canAllowMultiPleCurrency()
...
      *
      * @return boolean
      */
     public function allowCurrencySpeicifcCountry()
```
Comment (lizzyWithReach) [inline Helper/Data.php]: can you correct this typo too? allowCurrencySpecificCountry

Comment (BronzePointKrel) [reply Helper/Data.php]: No problem. There are spelling errors all over the place. Thankfully, most of them are in comments but sometimes (like this one) in function and method names.

File: Helper/Data.php [hunk 3]
```diff
@@ -86,15 +93,51 @@ public function getReachConfig($code, $storeId = null)
         return $this->getConfigValue(self::XML_PATH_REACH .'reach_gointerpay/'. $code, $storeId);
     }
 
-     /**
-     * Check Reach Enabled
+    /**
+     * Get Currency Options value REACH_CURRENCY_SWITCH - Function 06
```
Comment (lizzyWithReach) [inline Helper/Data.php]: out of curiosity, what's up with the Function numbers?

Comment (BronzePointKrel) [reply Helper/Data.php]: I'll do another update and remove that. I numbered them and then checked them off as I went through and tested each function. Wanted to make sure that I did a complete cart and purchase while watching those checkboxes.

**Review by lizzyWithReach — COMMENTED** [review]
Just had a couple of notes, if you could review the comments I've left

**Comment by lizzyWithReach** [comment]
You got all of the comments handled haha thank you! I'll merge this in

**Pull request merged**